slides/08-Platform-Hub/1050-YourAccount.md

```diff
@@ -0,0 +1,11 @@
+## Exploring your account
+
+Your platform hub account has some useful services examining:
+
+ - Terms of Service - Here we layout the terms of use relating to what is seen as acceptable practice on the acp platform. It's worth reading though since components have the ability the effect your projects work.
+
+ - Hub Setup - Here we layout the option of selecting Managerial and Technical roles. This will scope the hub to be more specific to your needs so as to stream line how interact with projects.
+
+ - Services Onboarding - Here we list all the ACP managed services you'll be able to access with your platform hub identity. Some are clickable and will link to homepage for each of those services (note: most are internal and so will need a vpn profile configured before accessing it)
```

Comment:
These 3 sections are all part of the onboarding flow on the hub, so maybe best to take them through it interactively instead?